explorer/app/shared/constants.ts

```diff
@@ -9,3 +9,12 @@ export const DEFAULT_DETAIL_PARAMS =
   config().datasources.defaultDetailParams ?? {};
 
 export const isDevelopment = () => process.env.NODE_ENV === "development";
+
+/**
+ * Values to determine the index for each param.
+ * https://host/explore/[slug]/[param-uuid]/[param-tab]
+ * Index 0 will return the current uuid
+ * Index 1 will return the current tab
+ */
```

Comment:
Can you rename these constants to give the namespace first? So, `PARAMS_INDEX_UUID` and `PARAMS_INDEX_TAB`?